package.json

```diff
@@ -277,10 +278,20 @@
     "redux-thunk": "^2.3.0",
     "source-map-support": "^0.5.9"
   },
+  "resolutions": {
+    "@octokit/rest":"16.33.1"
```

Comment:
Will we get issues with that or is it only used by flow-typed

Reply:
Shouldn't break